### PR TITLE
Remove quote in integration test queries to avoid folder name with quote

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Comment by shifucun** [comment]
To fix https://github.com/datacommonsorg/website/issues/4145

File: server/integration_tests/explore_test.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def test_detection_multivar(self):
         'compare obesity vs. poverty',
         'show me the impact of climate change on drought',
         'how are factors like obesity, blood pressure and asthma impacted by climate change',
-        'Compare "Male population" with "Female Population"',
+        'Compare Male population with Female Population',
```
Comment (pradh) [inline server/integration_tests/explore_test.py:264]: I think this is part of the test? (Multi-var handling tries to respect the double quotes)

Should we instead replace `"` in https://github.com/datacommonsorg/website/blob/cc3c3f29d3b8b13011295896c59c7bb381f7a290/server/integration_tests/explore_test.py#L91 ?

Comment (shifucun) [reply server/integration_tests/explore_test.py]: Done

File: server/integration_tests/explore_test.py [hunk 2]
```diff
@@ -61,7 +62,8 @@ def run_detection(self,
       if len(queries) == 1:
         d = ''
       else:
-        d = q.replace(' ', '').replace('?', '').lower()
+        d = re.sub(r'[ ?"]', '', q).lower()
+      print(d)
```
Comment (pradh) [inline server/integration_tests/explore_test.py:66]: Drop print?

**Review by pradh — APPROVED** [review]
Thanks!

**Pull request merged**